src/loaders/OMETiffLoader.js

```diff
@@ -95,7 +96,15 @@ export default class OMETiffLoader {
       this.omexml.getNumberOfImages() || (SubIFDs && SubIFDs.length);
     this.isBioFormats6Pyramid = SubIFDs;
     this.isPyramid = this.numLevels > 1;
-
+    // The omexml specification only allows for these - zarr is more flexible so this
+    // is for unifying the two loaders in upstream applications.
+    this.dimensions = [
+      { field: 'channel', type: 'nominal', values: this.channelNames },
+      { field: 'z', type: 'number', values: null },
+      { field: 'time', type: 'number', values: null },
```

Comment:
Should this be ordinal?  It has a maximum so I don't want people going overboard.  Should type 'number' have a maximum and a minimum?

Reply:
Right now we really just check for 'ordinal' or 'nominal', but the available types are from [vega-lite](https://vega.github.io/vega-lite/docs/type.html): `quantitative`, `temporal`, `ordinal`, and `nominal`. `number` isn't a type.